dune/gdt/interpolations/default.hh

```diff
@@ -46,26 +102,18 @@ namespace GDT {
  *
  * \note This might not be correct for all spaces, in particular if source is not continuous.
  */
-template <class GV, size_t r, size_t rC, class R, class V, class IGV>
-std::enable_if_t<std::is_same<XT::Grid::extract_entity_t<GV>, typename IGV::Grid::template Codim<0>::Entity>::value,
+template <class GV, size_t r, size_t rC, class R, class V, class IGVT>
```

Comment:
The ``IGVT`` is the ``Traits`` type, the ``IGV`` the actual ``GridView`` type. Without these changes, there were some errors because the compiler could not match ``Dune::GridView<IGV::Traits>`` and ``IGV``.